internal/hcp/api/client.go

```diff
@@ -92,55 +121,62 @@ func (c *Client) loadOrganizationID() error {
 }
 
 func (c *Client) loadProjectID() error {
+	if c.ProjectID != "" {
+		return nil
+	}
+	if env.HasProjectID() {
+		c.ProjectID = os.Getenv(env.HCPProjectID)
```

Comment:
I see that you removed the check that ensures the project exists, while this won't harm the normal use-case, I think it doesn't hurt to fail at that time rather than later with a more obscure message, is there a reason for that change?
Seeing the update, I presume this is a permission problem that prevents us from invoking `findProjectByID`?

Reply:
Hey there - so the permissions is exactly the reason for the removal. When using a plsp calling ListProjects will result in an error. It is not clear if this will always be true so opted not to make the call. 

I can see the added value of validating the project id. If possible I think find project by Id can be replaced with a validate Project ID method that makes a get request to the API to validate the project exist and has an active registry.